src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CFTTaskMapper.java

```diff
@@ -642,6 +652,36 @@ protected void reconfigureTaskAttribute(TaskResource taskResource,
         }
     }
 
+    protected void reconfigureAdditionalTaskAttribute(TaskResource taskResource,
+                                            List<ConfigurationDmnEvaluationResponse>
+                                                    configurationAdditionalAttributeDmnResponse) {
+        Map<String, String> existingAdditionalProperties = taskResource.getAdditionalProperties();
+        Map<String, Optional<String>> additionalProperties = new ConcurrentHashMap<>();
+
+        //Use optionals to be able to set null values
+        if (existingAdditionalProperties != null) {
+            existingAdditionalProperties.entrySet().forEach(
+                    e -> additionalProperties.put(e.getKey(), Optional.ofNullable(e.getValue())));
+        }
+
+        configurationAdditionalAttributeDmnResponse.stream().filter(response -> response.getCanReconfigure() != null
+                && response.getCanReconfigure().getValue()).forEach(
+                    response -> {
+                        if (response.getName().getValue() != null && response.getValue() != null) {
+                            additionalProperties.put(response.getName().getValue()
+                                    .replace(ADDITIONAL_PROPERTIES_PREFIX, ""),
+                                    Optional.ofNullable(response.getValue().getValue()));
+                        }
```

Comment:
We have decided to adhere what the DMN provides.  An empty string is maintained and stored as such in the DB.  This will also be the returned in the response in json if it is a field that's is returned.